spv/sync.go

```diff
@@ -1410,6 +1527,11 @@ func (s *Syncer) startupSync(ctx context.Context, rp *p2p.RemotePeer) error {
 		}
 	}
 
+	err = s.GetInitState(ctx, rp)
```

Comment:
This should only be called if `rp.Pver() >= wire.InitStateVersion`. The current peer code has a minimum supported version that is lower than the one that introduced the GetInitState message, so we could be connecting to a peer that does not support it.